modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -2605,22 +2605,6 @@ private void processSymbolizers(final Graphics2D graphics,
         }
     }
 
-    private Geometry clip(Geometry geometry, final GeometryClipper clipper,
-            boolean preserveTopology) {
-        try {
-            return clipper.clip(geometry, !preserveTopology);
```

Comment:
Yeah, by a cursory look it really looks like a mistake. Funny... I made several hand checks for it, must have been a last minute change.